fix: better outline with textShadow for subtitles

elpiel opened the pull request:
text shadow style has been taken from Stremio 4  and applied to both Html subtitles (external subtitles) and embedded.


### Embedded subtitles (after)
![image](https://github.com/user-attachments/assets/29d9ad3e-090f-48f4-b0bb-dbfbba2fac67)

### External subtitles (after)

![image](https://github.com/user-attachments/assets/385a7020-8ac4-4b93-85cd-24bf1c9147fe)

tymmesyde reviewed (commented):
`webkitTextStroke` should probably be used instead of `text-shadow` to achieve outline effect
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke

tymmesyde commented on src/withHTMLSubtitles/withHTMLSubtitles.js:
```diff
@@ -51,9 +51,19 @@ function withHTMLSubtitles(Video) {
+        var subsBgStyle = 1;
+        // Calculate the #c010r. First slice the # and cast to int
+        var intColor = parseInt(outlineColor.slice(1), 16);
+        var backgroundColor = 'rgba(' +
```
What is the purpose of this?

elpiel replied:
I took this from the stremio 4 desktop version. 

I believe it's meant to put some background which is based on the outline colour passed.

elpiel commented:
> `webkitTextStroke` should probably be used instead of `text-shadow` to achieve outline effect
> https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke

Ok, how can we add the WebKit version?
Maybe we could merge this first and then tweak it as now it does affect reading subtitles on production

elpiel commented:
Due to webkitTextStroke not being supported in `::cue` (https://developer.mozilla.org/en-US/docs/Web/CSS/::cue) I've left the text shadow.

Other notable changes:
- Updated the Embedded subs. styling
- updated the `em` units to `rem`
- tested out other shadow options
